tensorflow/docs_src/extend/adding_an_op.md

```diff
@@ -341,9 +350,9 @@ Assuming you have `g++` installed, here is the sequence of commands you can use
 to compile your op into a dynamic library.
 
 ```bash
-TF_CFLAGS=( $(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_compile_flags()))') )
```

Comment:
Why did you change these flags?

Reply:
My bad. I mixed up which ones were newer; just realized I was compiling with 1.0 instead of 1.5. I'll change them back.